core/domain/fs_services_test.py

```diff
@@ -87,58 +84,6 @@ def test_save_original_and_compressed_versions_of_image(self):
         self.assertEqual(
             fs.isfile('image/%s' % self.MICRO_IMAGE_FILENAME), True)
 
-    def test_compress_image_on_prod_mode_with_big_image_size(self):
```

Comment:
Why are you deleting this? In general, never delete existing tests -- at least have an equivalent test that replaces it.

Reply:
There is an identical test below it besides one small change which is adding the max resize dimension. As we mentioned before, restricting image resize to < 4000 x 4000 is obsolete since PIL doesn't have that restriction. Since the functionality is the same except for this one part, this test is now obsolete.